bitnami/common/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
 type: library
-version: 2.13.0
+version: 2.13.1
```

Comment:
Could you please bump a new version? 

Version 2.13.1 has already been released: https://github.com/bitnami/charts/blob/main/bitnami/common/Chart.yaml

Reply:
```suggestion
version: 2.14.0
```

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
>    MAJOR version when you make incompatible API changes
>    __MINOR version when you add functionality in a backward compatible manner__
>    PATCH version when you make backward compatible bug fixes

Reply:
@jouve This is not really a new feature such as adding a new helper, but a patch for an existing one

Reply:
Thanks @migruiz4 , @jouve .  I've updated to 2.13.2.